core/src/color.rs

```diff
@@ -183,48 +183,48 @@ macro_rules! color {
 }
 
 #[cfg(feature = "palette")]
-/// Converts from palette's `Srgba` type to a [`Color`].
-impl From<Srgba> for Color {
-    fn from(srgba: Srgba) -> Self {
-        Color::new(srgba.red, srgba.green, srgba.blue, srgba.alpha)
+/// Converts from palette's `Rgba` type to a [`Color`].
+impl From<Rgba> for Color {
+    fn from(rgba: Rgba) -> Self {
+        Color::new(rgba.red, rgba.green, rgba.blue, rgba.alpha)
```

Comment:
Why has this been changed? `Color` is in the sRGB color space.

Reply:
Ah, I see. It's a type definition! Still, it seems more descriptive to use `Srgb` directly.

Reply:
Reverted it in b741893013eda90045e5a3950c37ba21d073b351 for now.